force-app/main/default/lwc/daysOnMarket/daysOnMarket.css

```diff
@@ -28,9 +24,11 @@
 }
 
 .days {
+    /* @sldsValidatorIgnore */
```

Comment:
It might be good to add a comment that explain why we chose to ignore these validation warnings.